mobile/lib/modules/backup/views/backup_controller_page.dart

```diff
@@ -48,7 +48,7 @@ class BackupControllerPage extends HookConsumerWidget {
         }
 
         if (Platform.isIOS) {
-          ref.read(iOSBackgroundSettingsProvider).refresh();
+          ref.read(iOSBackgroundSettingsProvider.notifier).refresh();
```

Comment:
You probably want to use `ref.watch` here as well?

Reply:
I thought you weren't supposed to put `ref.watch` in a `useEffect` block. I just wanted a place to refresh the data when we load this page.

You said:
> Riverpod state that you should not user ref.watch in any async/initialization/button press, user ref.read instead
![image](https://user-images.githubusercontent.com/100457/220986351-f7b478ab-3383-48c6-9d03-1a42af465f08.png)

And I thought `useEffect` was an initialization? But I may be misunderstanding, because there is a `watch` right above it in this block. Happy to change it to a `watch`.

Reply:
Either way, I changed this to `watch`. Thanks!